js/data.json

```diff
@@ -1471,5 +1471,25 @@ osts = [
 		"link": "https://www.dropbox.com/s/adhvu2r1646c2j0/Gohan%20ya-san%20Megumi.mp3?dl=1",
 		"name": "Amaama to Inazuma OST - Gohan ya-san Megumi",
 		"img": "https://www.dropbox.com/s/suvtvzcdm5rptxy/amaama%204.jpg?dl=1"
+	},
+	{
+		"link": "https://www.dropbox.com/s/obccd111g0a4ho3/28%20-%20Samidare.mp3?dl=0",
+		"name": "Naruto OST - Samidare",
+		"img": "https://www.dropbox.com/s/c36rmzzdmkvpw54/jiraya_img.jfif?dl=0"
+	},
+	{
+		"link": "https://www.dropbox.com/s/uo9kqpcw2mz470t/Demon%20SlayerOPGurenge%20by%20LiSA%20HD.mp3?dl=0",
+		"name": "Kimetsu No Yaiba - OP ",
```

Comment:
You should add the name of the Op, same with all songs, like you did with Naruto

`Kimetsu No Yaiba - OP: Gurenge`

Reply:
So can I call the ep 19 ending as kamaboko gonpanchiro no uta? just kidding. I've made the changes as requested.

Reply:
🤣 🤣 🤣